CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_xero v0.7.1
+[PR #54](https://github.com/fivetran/dbt_xero/pull/54) includes the following updates:
+
+## Under the Hood
```

Comment:
I would actually phrase this as a bugfix since this will have an impact on the model count if the tables are selected in the schema tab and present in their destination. As such, let's define this as a bugfix and reword this to ensure the users are aware of the change this may have on their model count.